agnpy/spectra/spectra.py

```diff
@@ -198,6 +200,42 @@ def plot(self, gamma=None, gamma_power=0, ax=None, **kwargs):
 
         return ax
 
+    def evaluate_time(self, time, energy_loss_function, subintervals_count=1):
```

Comment:
I would set by default the subintervals_count to a higher number, maybe 10

Reply:
done